testdata/operations/valid-operation.json

```diff
@@ -1,18 +1,20 @@
 {
-    "installation_name": "test",
-    "revision": "01DDY0MT808KX0GGZ6SMXN4TW",
-    "action": "install",
-    "parameters": {
-        "param1": "value1",
-        "param2": "value2"
-    },
+  "installation_name": "test",
+  "revision": "01DDY0MT808KX0GGZ6SMXN4TW",
+  "action": "install",
+  "parameters": {
+    "param1": "value1",
+    "param2": "value2"
+  },
+  "image": {
     "image": "testing.azurecr.io/duffle/test:e8966c3c153a525775cbcddd46f778bed25650b4",
```

Comment:
This image needs auth to pull, otherwise I'd like to add a digest to the test file.

Reply:
Updated to use a public sample invocation image in https://github.com/deislabs/cnab-go/pull/114/commits/3ad2f2446805c4b3c02601a3df52ad284136f8fb